os/running-rancheros/server/pxe/index.md

```diff
@@ -11,11 +11,11 @@ layout: os-default
 # Boot a persistent RancherOS to RAM
 
 # Location of Kernel/Initrd images
-$ set base-url http://releases.rancher.com/os/latest
+set base-url http://releases.rancher.com/os/latest
 
-$ kernel ${base-url}/vmlinuz rancher.state.autoformat=[/dev/sda] rancher.cloud_init.datasources='["url:http://example.com/cloud-config"]'
```

Comment:
This kernel parameter **must** use this quoting syntax: 
```
rancher.cloud_init.datasources='["url:http://example.com/cloud-config"]'
```

Kernel parameters like this (corresponding to ros config keys) are supposed to be in YAML syntax. We need quotes around the URL, so we better have some way of passing them without escaping. Luckily, the value **inside** single quotes gets passed as is (preserving the enclosed quotes).